index.html

```diff
@@ -85,10 +91,10 @@ <h2 class="head-description">Multy-Post Stories</h2>
                     <li class="item">JavaScript</li>
```

Comment:
Kindly implement these changes for the navigation and headline section of your portfolio :

- the logo text should be on the same vertical line with `Hey there, I'm Alex` text similar to what we have in our Figma design 😄 
- The background image and the `headline` section seem to be a bit trimmed from the down part as it doesn't contain the full height of the section

Figma design :
![image](https://github.com/AndresEspi/portafolio/assets/20537712/0d10b667-35a8-48ed-a97f-468854099a68)

Your design :

![image](https://github.com/AndresEspi/portafolio/assets/20537712/2b2ffbd2-483b-4229-90bb-c9f0dd7e084f)